frontend/src/App.tsx

```diff
@@ -66,15 +66,26 @@ function App() {
   return (
     <>
       {/* Notice bar */}
-      <Notice
-        // Increment for each new notice (though you don't need to change it
-        // when removing a notice), or users who previously dismissed the banner
-        // won't see the updated content.
-        id={4}
-      >
-        Basic course information is now publicly available without login! Share
-        courses with your family and friends with ease ;)
-      </Notice>
+      {location.pathname === '/catalog' && authStatus === 'unauthenticated' ? (
+        <Notice
+          // Increment for each new notice (though you don't need to change it
+          // when removing a notice), or users who previously dismissed the banner
+          // won't see the updated content.
+          key="unauthenticated-notice"
+          id={5}
+        >
```

Comment:
Don't use a notice because it's closeable and then will never reopen. Use a separate dialog box in the lower right corner. See also https://github.com/coursetable/coursetable/issues/1633

Reply:
@benzuzu Thoughts on taking over? I'm kinda scuffed when it comes to frontend :)

Reply:
@Josh-Cena, I've thought about this more and I think we should replace the Me icon with a primary colored sign in button for unauthenticated access - something very similar to the approach google.com takes.

As for #1633, I think the exclamation point in the Me icon with an action item in the dropdown is the way to go.

Reply:
Yep agreed for the first part. See my previous slack message:

> (For logged in users) I think it should show name initials instead of the default icon here, and we should have a way to let people customize their name in case there's no Yalies data, they have diff preferred name, etc.

But I also like your idea of a primary colored button.